atlasdb-api/src/main/java/com/palantir/atlasdb/spi/TransactionKeyValueServiceManagerFactory.java

```diff
@@ -38,6 +38,8 @@ public interface TransactionKeyValueServiceManagerFactory<T> {
     Class<T> coordinationValueClass();
 
     TransactionKeyValueServiceManager create(
+            String namespace,
+            // TODO(jakubk): Add user agent, dialogue factory etc.
```

Comment:
@jeremyk-91 any objections?

Reply:
Discussed offline: These are for sharing of Dialogue resources and observability, which makes sense. Yeah, no objections here

Reply:
Just documenting the consequences: some conjure (for UserAgent) APIs and dialogue APIs (for client factories) will now be an explicit dependency of atlasdb-api.

They have always been in atlasdb-clients etc., but not here. If this is a concern, we would probably need an explicit separate API jar or something